src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -55,6 +55,11 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
+            services.AddHostFiltering(options =>
+            {
+                options.AllowedHosts = Configuration.GetSection("AllowedHosts").Get<List<string>>();
```

Comment:
Will `Bind` not work here? Could we do an overload of `AddHostFiltering` that takes the configuration section directly?

Reply:
Updated to use Bind